test/v1/models/response/retrieveDisclosures/RetrieveDisclosuresResponseSpec.scala

```diff
@@ -59,22 +58,6 @@ class RetrieveDisclosuresResponseSpec extends UnitSpec {
       }
     }
 
-    "read from valid JSON with empty taxAvoidance array" should {
-      "produce an empty RetrieveDisclosuresResponse object" in {
-        val json = Json.parse(
-          """
-            |{
-            |   "taxAvoidance": [ ],
-            |   "submittedOn": "2020-07-06T09:37:17Z"
-            |}
-          """.stripMargin
-        )
-
-        json.as[RetrieveDisclosuresResponse] shouldBe
-          RetrieveDisclosuresResponse.empty.copy(submittedOn = "2020-07-06T09:37:17Z")
-      }
-    }
-
```

Comment:
This scenario shouldn't be removed. It's still valid.

Reply:
Yes, it is valid but wasn't sure it was necessarily needed as we didn't do this test for most of the newer endpoints. It was removed because it used **.empty** which was removed from the RetrieveDisclosuresResponse file as submittedOn is mandatory. If you want me to add the test, I can though, just wasn't sure it was necessarily needed.

Reply:
The functionality it tests is written by us and not part of the auto-generated Play JSON reads so it needs testing.

Reply:
No worries. Will write a test for it. Thanks